docs/USAGE.md

```diff
@@ -0,0 +1,114 @@
+```hcl
+source "macstadium-orka" "image" {
+  source_image    = "90GBigSurSSH.img"
+  image_name      = "BigSurGolden.img"
+  orka_endpoint   = "http://10.221.188.100"
+  orka_user       = "<USER_NAME>"
+  orka_password   = "<PASSWORD>"
+  ssh_username    = "<SSH_USERNAME>"
+  ssh_password    = "<SSH_PASSWORD>""
+}
+```
+The source definition specifies a plugin type to use (in this case “macstadium-orka”) and a label associated with the type ( image”).   This is saying that these variables will   
+be passed to the macstadium-orka plugin with the label image.   Adding plugin blocks with additional labels allows for parallel builds. The variables passed in here are recommended   
+for the orka packer plugin to function.  The only required variables are `source_image`,  `orka_user`,  `orka_password` and `orka_endpoint`.  For more information on these please view  
+the [Configuration Doc](./builders/config.mdx)   
+
+The build section specifies the type of build you want to perform and the type actions to take.  
+
+```hcl
+build {
+  sources = [
+    "macstadium-orka.image"
+  ]
+   provisioner "shell" {
+    inline = [
+      "echo we are running on the remote host",
+      "hostname",
+      "touch .we-ran-packer-successfully"
+    ]
+  }
+}
+```
+
+The sources array specifies the plugin source to use.  You can run different builds in parallel.  The provisioner block is what’s responsible for running commands in the created virtual machine.  
+Once an ssh connection is established, a commands in the inline array block will be executed.  In the case of the above snippet, an `echo` and `hostname` command will execute as well as creating   
+a file called .we-ran-packer-successfully.  Once this completes, the plugin will save a new image.  
+
+Putting it all together, you should have file looking like this.   
+
+```hcl
+packer {
+  required_plugins {
+    macstadium-orka = {
+      version = ">= 2.3.0"
+      source  = "github.com/macstadium/macstadium-orka"
+    }
+  }
+}
+source "macstadium-orka" "image" {
+  source_image    = "90GBigSurSSH.img"
+  image_name      = "BigSurGolden.img"
+  orka_endpoint   = "http://10.221.188.100"
+  orka_user       = "<USER_NAME>"
+  orka_password   = "<PASSWORD>"
+  ssh_username    = "<SSH_USERNAME>"
+  ssh_password    = "<SSH_PASSWORD>""
+}
+build {
+  sources = [
+    "macstadium-orka.image"
+  ]
+   provisioner "shell" {
+    inline = [
+      "echo we are running on the remote host",
+      "hostname",
+      "touch .we-ran-packer-successfully"
+    ]
+  }
+}
+```
+
+Save this as file named orka.pkr.hcl.  You can actually call this any name you want to but you have to make sure the file ends in the suffix pkr.hcl or else it will not work.    
+
+To initialize and download the plugin use this command:  
+`packer init orka.pkr.hcl`  
+
+If everything worked correctly, you should see output similar to this:  
+![Packer Init](../images/packerinit.PNG)
+
+To build your image, enter:  
```

Comment:
Maybe we need a step before this one "Ensure you are connected your Orka cluster via VPN"

Reply:
That's a good idea. Done